frontendUiTable/src/main/java/com/virtuslab/gitmachete/frontend/ui/table/GitMacheteGraphTableManager.java

```diff
@@ -60,6 +64,13 @@ public GitMacheteGraphTableManager(Project project,
     subscribeToGitRepositoryChanges();
   }
 
+  @SneakyThrows
```

Comment:
I'm losing my patience to checked exceptions ;p https://projectlombok.org/features/SneakyThrows

Reply:
Btw later JVM languages (like Scala) ditched checked exceptions completely (in Scala, `Either` is favored for that purpose)